<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CHyprlandWelcome</class>
 <widget class="QMainWindow" name="CHyprlandWelcome">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>CHyprlandWelcome</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../Documents/Hypr/try3finaltransparent.png</normaloff>../../Documents/Hypr/try3finaltransparent.png</iconset>
  </property>
  <property name="windowOpacity">
   <double>0.900000000000000</double>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="logo">
    <property name="geometry">
     <rect>
      <x>315</x>
      <y>0</y>
      <width>160</width>
      <height>90</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="Logo.qrc">:/assets/assets/logobanner.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabs">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>780</width>
      <height>371</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <property name="usesScrollButtons">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="welcomeTab">
     <attribute name="title">
      <string>Welcome</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>-1</x>
        <y>10</y>
        <width>778</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;Welcome to Hyprland!&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>751</width>
        <height>211</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;We hope you enjoy your stay. In order to help you get accomodated to Hyprland in an easier manner, we prepared a little basic setup tutorial, just for you.&lt;/p&gt;&lt;p&gt;If you feel adventurous, or are an advanced user, you can click the &amp;quot;don't show again&amp;quot; button on the bottom. It will close this window and never show it again.&lt;/p&gt;&lt;p&gt;If you want to manually launch this app, just execute HyprlandWelcome in your terminal.&lt;/p&gt;&lt;p&gt;Click the &amp;quot;next&amp;quot; button to proceed to the next step of our setup :)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>50</y>
        <width>216</width>
        <height>20</height>
       </rect>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="tab1Next">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>315</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string>Next</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="gettingStartedTab">
     <attribute name="title">
      <string>Getting Started</string>
     </attribute>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>751</width>
        <height>161</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The first thing we'll need to do is get some packages installed that you absolutely need in order for your system to be working properly.&lt;/p&gt;&lt;p&gt;Apps with * are absolutely necessary. All other are &lt;span style=&quot; font-weight:700; font-style:italic;&quot;&gt;highly&lt;/span&gt; recommended, as they provide core parts of a working environment.&lt;/p&gt;&lt;p&gt;You &lt;span style=&quot; font-style:italic;&quot;&gt;can &lt;/span&gt;proceed without any of those, but it's not advised. There is a possibility that this app is unable to detect some of your installed binaries. In that case, it's okay to ignore them.&lt;/p&gt;&lt;p&gt;Use the &amp;quot;launch terminal&amp;quot; button to launch a terminal. Supported terminals are kitty, alacritty, foot, wezterm, konsole, gnome-terminal, xterm. If none are installed, you'll need to exit hyprland and install one. Use SUPER+M to exit hyprland.&lt;br/&gt;&lt;br/&gt;Hover on the different components to see what options are accepted.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="INSTALL_TERM">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>180</y>
        <width>301</width>
        <height>21</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Accepted: kitty, alacritty, wezterm, foot, konsole, gnome-terminal. Worth noting this is not a comprehensive list and Hyprland will work with any wayland-compatible terminal emulator, however this app will not automatically detect it.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;❌&lt;/span&gt; Terminal&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="INSTALL_FM">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>200</y>
        <width>301</width>
        <height>21</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Accepted: dolphin, ranger, thunar&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;❌&lt;/span&gt; File Manager&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="INSTALL_WALLPAPER">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>220</y>
        <width>301</width>
        <height>21</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Accepted: hyprpaper, swaybg, swww, wpaperd. Recommended: hyprpaper&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;❌&lt;/span&gt; Wallpaper Utility&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="INSTALL_PW">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>240</y>
        <width>301</width>
        <height>21</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Accepted: pipewire &lt;span style=&quot; font-weight:700;&quot;&gt;and&lt;/span&gt; wireplumber&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;❌&lt;/span&gt; Pipewire*&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="INSTALL_XDP">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>180</y>
        <width>301</width>
        <height>21</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Accepted: xdg-desktop-portal &lt;span style=&quot; font-weight:700;&quot;&gt;and&lt;/span&gt; xdg-desktop-portal-hyprland. Recommended to also install xdg-desktop-portal-gtk for a file picker.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;❌&lt;/span&gt; XDG Desktop Portal*&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="INSTALL_AUTH">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>200</y>
        <width>301</width>
        <height>21</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Accepted: &lt;span style=&quot; font-family:'monospace';&quot;&gt;kde-authentication-agent&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;❌&lt;/span&gt; Authentication Agent&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="INSTALL_QTW">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>220</y>
        <width>301</width>
        <height>21</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Accepted: qt5-wayland, qt6-wayland (highly recommended both)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;❌&lt;/span&gt; QT Wayland support&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="INSTALL_NOTIF">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>280</y>
        <width>301</width>
        <height>21</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Accepted: dunst, mako. Worth noting you can also create your own notification daemons with stuff like ags or eww.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;❌&lt;/span&gt; Notification Daemon&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="INSTALL_BAR">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>260</y>
        <width>301</width>
        <height>21</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Accepted: waybar, eww, ags&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;❌&lt;/span&gt; Status Bar&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="INSTALL_LAUNCHER">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>260</y>
        <width>301</width>
        <height>21</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Accepted: fuzzel, wofi, anyrun, tofi&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;❌&lt;/span&gt; App Launcher&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="tab2Next">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>315</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string>Next</string>
      </property>
     </widget>
     <widget class="QPushButton" name="openTerminalButton">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>315</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string>Open Terminal</string>
      </property>
     </widget>
     <widget class="QLabel" name="INSTALL_QTTHEME">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>240</y>
        <width>301</width>
        <height>21</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Accepted: qt5ct, qt6ct.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;❌&lt;/span&gt; QT Theming&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="configuringTab">
     <attribute name="title">
      <string>Configuring</string>
     </attribute>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>751</width>
        <height>241</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Now it's time to configure the apps you've installed.&lt;/p&gt;&lt;p&gt;Status bars, wallpaper utilities might require their own configs. See their github pages / wikis to gain info on how to.&lt;/p&gt;&lt;p&gt;Open the config located at ~/.config/hypr/hyprland.conf and edit:&lt;/p&gt;&lt;p&gt;At the top, add exec-once = name for your autostart apps &lt;span style=&quot; font-style:italic;&quot;&gt;(e.g. exec-once = hyprpaper)&lt;/span&gt;&lt;/p&gt;&lt;p&gt;A bit below that, find the default variables (e.g. $terminal) and edit them to your choices.&lt;/p&gt;&lt;p&gt;Further down, you'll see them used in bind = commands. You can edit those binds to your likings.&lt;/p&gt;&lt;p&gt;That's it! Your basic configuration is done. Head onto &lt;a href=&quot;https://wiki.hyprland.org/Configuring/Configuring-Hyprland/&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#8ab4f8;&quot;&gt;the wiki&lt;/span&gt;&lt;/a&gt; to learn more about how to properly configure Hyprland. &lt;span style=&quot; font-style:italic;&quot;&gt;(this link might not open, in which case use the big button or manually go to https://wiki.hyprland.org/)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="tab3Next">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>315</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string>Next</string>
      </property>
     </widget>
     <widget class="QPushButton" name="openTerminalButton2">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>315</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string>Open Terminal</string>
      </property>
     </widget>
     <widget class="QPushButton" name="openWikiButton">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>315</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string>Open Wiki</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="closingNotesTab">
     <attribute name="title">
      <string>Closing Notes</string>
     </attribute>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>751</width>
        <height>241</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Thank you for choosing to install Hyprland. Have fun!&lt;/p&gt;&lt;p&gt;Make sure to check out the wiki, it documents pretty much everything Hyprland has to offer in detail.&lt;/p&gt;&lt;p&gt;The Variables page will document all the toggleable options, other pages will document stuff like monitor configs, binds, execs, sourcing, etc.&lt;/p&gt;&lt;p&gt;By clicking &amp;quot;Finish&amp;quot; this window will not pop up ever again. If you click &amp;quot;exit&amp;quot;, it will pop up again next time you open Hyprland.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="finishButton">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>315</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string>Finish</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="exitButton">
    <property name="geometry">
     <rect>
      <x>699</x>
      <y>470</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dontShowAgain">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>470</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>Don't show this again :(</string>
    </property>
   </widget>
   <widget class="QPushButton" name="githubButton">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>470</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>GitHub</string>
    </property>
   </widget>
   <widget class="Line" name="bottomHr">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>780</width>
      <height>1</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Light">
        <brush brushstyle="SolidPattern">
         <color alpha="200">
          <red>156</red>
          <green>156</green>
          <blue>156</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>166</red>
          <green>166</green>
          <blue>166</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>176</red>
          <green>176</green>
          <blue>176</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Light">
        <brush brushstyle="SolidPattern">
         <color alpha="200">
          <red>156</red>
          <green>156</green>
          <blue>156</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>166</red>
          <green>166</green>
          <blue>166</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>176</red>
          <green>176</green>
          <blue>176</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Light">
        <brush brushstyle="SolidPattern">
         <color alpha="200">
          <red>156</red>
          <green>156</green>
          <blue>156</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>176</red>
          <green>176</green>
          <blue>176</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>176</red>
          <green>176</green>
          <blue>176</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="Logo.qrc"/>
 </resources>
 <connections/>
</ui>
